pkg/cable/fake/driver.go

```diff
@@ -54,6 +55,10 @@ func (d *Driver) Init() error {
 	return d.ErrOnInit
 }
 
+func (d *Driver) SetupNATDiscovery(natDiscovery natdiscovery.Interface) {
+
+}
```

Comment:
This isn't needed in the `Driver`.

```suggestion
```

Reply:
oh, right, why did I put it in there? :?

Reply:
Done